app/dashboard/views/custom_alias.py

```diff
@@ -174,41 +175,29 @@ def verify_prefix_suffix(user, alias_prefix, alias_suffix) -> bool:
 
     # make sure alias_suffix is either .random_word@simplelogin.co or @my-domain.com
     alias_suffix = alias_suffix.strip()
-    if alias_suffix.startswith("@"):
```

Comment:
This code part is indeed overly complex as it needs to take into account the self-hosting case ... I think the new code misses the check on the `alias_domain_prefix` though as we want to make sure users can't inject any string here. I wrote a small gist for it: https://github.com/simple-login/app/commit/43857495b3da2c99e5ee51f0a52e51bc3d55f9d8, let me know if this makes sense

Reply:
It mostly makes sense but you seem to have removed the DISABLE_ALIAS_PREFIX check? Why is that?

Reply:
Yeah when re-writing this part, it seems that there's no need to distinguish the `DISABLE_ALIAS_PREFIX` case. I guess this case has been added as a hotfix back then 🤔.